contracts/contracts/utils/InitializableAbstractStrategy.sol

```diff
@@ -82,6 +82,9 @@ abstract contract InitializableAbstractStrategy is Initializable, Governable {
      * @param _config The platform and OToken vault addresses
      */
     constructor(BaseStrategyConfig memory _config) {
+        // Nobody owns the implementation
+        _setGovernor(address(0));
+
```

Comment:
Looks great.

We need to make sure the unit tests are okay with this.

I see one error

```
  1) OETH Vault
       Config
         Fail to cacheWETHAssetIndex if WETH is not a supported asset:
     AssertionError: Expected transaction to be reverted with "Invalid WETH Asset Index", but other reason was found: "Caller is not the Governor"
```